Bottom: 1066e5cc20b35d7f3c6e4ea656deb3c623fd15d9
Top:    c429a6144a04967f9e6dfe112b317d869692e1a9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 01:10:53 +0100

Ensure "Gathering" mode is set after painting the screen.



---

diff --git a/src/draw.c b/src/draw.c
index 8322879..f710567 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2711,4 +2711,6 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
   Output.fgGC = savefg;
   Output.bgGC = savebg;
   Output.pmGC = savepm;
+
+  Gathering = True;
 }
